packages/mdc-select/mdc-select.scss

```diff
@@ -175,8 +163,6 @@ $mdc-select-menu-transition: transform 180ms $mdc-animation-standard-curve-timin
   // JS-enhanced and CSS-only Selects require different selectors for focused bottom-line due to different DOM structure
   &__surface:focus .mdc-select__bottom-line,
   &__surface:focus ~ .mdc-select__bottom-line {
```

Comment:
ah...see my comment above where I was confused why we need this.

I would move this to another mixin like this

```
@mixin mdc-select-focused-bottom-line {
// JS-enhanced and CSS-only Selects require different selectors for focused bottom-line due to different DOM structure
  .mdc-select__surface:focus .mdc-select__bottom-line,
  .mdc-select__surface:focus ~ .mdc-select__bottom-line {
    @content;
  }
}
```

And then use that mixin here in mdc-select.scss and in mdc-select/_mixins.scss